src/olympia/hero/migrations/0007_auto_20200603_0207_squashed_0010_primaryhero_select_image.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.2.12 on 2020-06-15 18:04
+
+from django.db import migrations, models
+import django.db.models.deletion
+import olympia.hero.models
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('hero', '0007_auto_20200603_0207'), ('hero', '0008_remove_primaryhero_custom_image'), ('hero', '0009_primaryheroimage'), ('hero', '0010_primaryhero_select_image')]
```

Comment:
Upon removing the `replaces` and `AlterField` lines, I had to update a migration file in discovery, which had a dependency on the 0009 hero migration file when I had added the 'Primary hero images' under the Discovery section in Admin. 

Local state of the database doesn't seem to be acting weird?? ... yet. I'll keep you posted.

Reply:
the weirdness might happen after you delete migrations 0008 to 0010 and then try to create a new migration.

Reply:
After squashing the migrations, I checked `showmigrations` which showed it was not utilizing 0007-0010 after the squash, so I removed those migration files. 

When I went ahead to make the migration to add the `image` property back into the fields in the PrimaryHero model, the migration had added the AlterField lines back in that were mentioned to be removed below, which I took out again before committing. No weirdness yet... but I'll let you know in case I just jinxed myself.